Sources/CCFilter/Models/ViewModels/RangeFilterInfo.swift

```diff
@@ -2,7 +2,7 @@
 //  Copyright © FINN.no AS, Inc. All rights reserved.
 //
 
-public final class RangeFilterInfo {
+public class RangeFilterInfo {
     typealias RangeBoundsOffsets = (hasLowerBoundOffset: Bool, hasUpperBoundOffset: Bool)
     typealias AccessibilityValues = (stepIncrement: Int?, valueSuffix: String?)
     typealias AppearenceProperties = (usesSmallNumberInputFont: Bool, displaysUnitInNumberInput: Bool, isCurrencyValueRange: Bool)
```

Comment:
Instead of subclassing I'd better add a property `let kind: Kind`, which is an enum:

```swift
public final class RangeFilterInfo {
   public enum Kind {
      case slider
      case stepper
   }

   init(kind: Kind, ...) { ... }
} 
```

Reply:
Yes! Good idea 😄